sig-architecture/production-readiness.md

```diff
@@ -80,9 +80,28 @@ To become a reviewer:
  * Read/study previous PRR comments and production readiness responses in existing KEPs.
  * Choose some KEPs requiring PRR and perform a review. Put "shadow prod readiness review"
    in your review comments so that the assigned PRR approver knows your intent.
- * After at least one release cycle, if you have shown good judgement and quality reviews,
-   you can propose yourself as approver by submitting a PRR to add your GitHub
-   handle to the `prod-readiness-approvers` alias in [OWNERS_ALIASES].
+
+### Becoming an approver
+
+After serving as reviewer/shadow for at least one release and showing good judgement and quality reviews,
+you can propose yourself as an approver by submitting a PR to add your GitHub
+handle to the `prod-readiness-approvers` alias in [OWNERS_ALIASES].
+
+When submitting the PR, you should include references to KEPs you reviewed that demonstrated a good variety
+of different situations.
+Here is a good starting point (remember that one PR can cover multiple categories):
+
+* Transitions from new to alpha
+* Transitions from alpha to beta
+* Transitions from beta to GA
+* Must have successfully reviewed at least three enhancements that require coordination between multiple components.
```

Comment:
What's the criteria for success?

Reply:
It's this line and right underneath.

Reply:
Oh misunderstood. I guess we need the checkmark from our secondary reviewer.

Reply:
My intuition is that success would be along the lines of:  The primary reviewer felt that the PRR review performed by the shadow reviewer was of sufficient quality that the primary review wasn't essential. But I'm curious what others think.

Reply:
That's how I'm thinking about it.

@deads2k @johnbelamaric - thoughts?

Reply:
Yes, agree.